modules/nw-multus-configuring-whereabouts-ip-reconciler-schedule.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="nw-multus-configuring-whereabouts-ip-reconciler-schedule_{context}"]
+= Configuring the Whereabouts IP reconciler schedule
+
+The Whereabouts IPAM CNI plugin runs the IP reconciler daily to clean up any stranded IP allocations that might result in exhausting IPs and therefore prevent new Pods from getting an IP allocated to them.
+
+Use this procedure to change the frequency at which the IP reconciler runs. 
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have deployed the `whereabouts-reconciler` daemon set, and the `whereabouts-reconciler` pods are up and running.
+
+.Procedure
+
+. Run the following command to create a `ConfigMap` named `whereabouts-config` in the `openshift-multus` namespace with a specific cron expression for the IP reconciler:
++
+[source,terminal]
+----
+$ oc create configmap whereabouts-config -n openshift-multus --from-literal=reconciler_cron_expression="*/15 * * * *"
+----
++
+This cron expression indicates the IP reconciler runs every 15 minutes. Adjust the expression based on your specific requirements.
++
+[NOTE]
+====
+The `whereabouts-reconciler` can only consume a cron expression pattern that includes five asterisks. The sixth, which is used to denote seconds, is currently not supported.
```

Comment:
Is there something more specific to call this here?
```suggestion
The `whereabouts-reconciler` daemon set can only consume a cron expression pattern that includes five asterisks. The sixth, which is used to denote seconds, is currently not supported.
```
Edit after making a few other related comments: You could also switch these naked `whereabouts-reconciler` instances to more generally "the Whereabouts reconciler", which I see is used in the other procedure.